build/Build.cs

```diff
@@ -42,7 +42,7 @@ partial class Build : NukeBuild
 
     [Solution] readonly Solution Solution;
     [GitRepository] readonly GitRepository GitRepository;
-    [GitVersion] readonly GitVersion GitVersion;
+    [GitVersion(Framework = "net6.0")] readonly GitVersion GitVersion;
```

Comment:
GitVersion support multiple .NET version, `Framework` tells it what .NET version to use